pkg/sidecar-controller/groupsnapshot_controller_test.go

```
@@ -0,0 +1,1463 @@
+			if (err != nil) != tt.wantErr {
+				t.Errorf("getCSIGroupSnapshotInput() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if tt.wantErr && tt.errSubstring != "" && (err == nil || !containsSimple(err.Error(), tt.errSubstring)) {
+				t.Errorf("getCSIGroupSnapshotInput() error = %v, want substring %q", err, tt.errSubstring)
+			}
+			if !tt.wantErr {
+				if (class != nil) != tt.wantClass {
+					t.Errorf("getCSIGroupSnapshotInput() class = %v, wantClass %v", class != nil, tt.wantClass)
+				}
+				if creds == nil && tt.wantClass {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)